lib/HTTP/Cookies/Microsoft.pm

```diff
@@ -1,6 +1,8 @@
 package HTTP::Cookies::Microsoft;
 
 use strict;
+use warnings;
+use vars qw(@ISA);
```

Comment:
Why's the legacy `vars` pragma being introduced? `@ISA` is declared correctly with `our` a few lines down. Though something like `parent` might be better still it's out of the scope of this PR.